.gitignore

```diff
@@ -51,6 +51,12 @@ static/riffraff
 
 static/src/stylesheets/icons/*
 
+# cse ( https://github.com/jeteve/App-CSE/ ) #
```

Comment:
I'm not sure this should have been committed?

Reply:
Looks like a change specific to you and could be applied to your global `.gitignore`.

Reply:
global .gitignore? Cool!